src/components/molecules/UiLoader/_internal/UiLoaderTransition.vue

```diff
@@ -123,7 +125,7 @@ const startTransition = () => {
   }
   onTransitionBeforeActive();
 };
-onMounted(() => {
+onMounted(async () => {
```

Comment:
Why async if we're not awaiting anything?

Reply:
typo; thanks